packages/sampler/stories/default/ViewManager.js

```diff
@@ -0,0 +1,134 @@
+import {action} from '@enact/storybook-utils/addons/actions';
+import {boolean, number, select} from '@enact/storybook-utils/addons/knobs';
+import {mergeComponentMetadata} from '@enact/storybook-utils';
+import Group from '@enact/ui/Group';
+import Item from '@enact/ui/Item';
+import Layout, {Cell} from '@enact/ui/Layout';
+import Scroller from '@enact/ui/Scroller';
+import {arrange, SlideArranger, SlideBottomArranger, SlideLeftArranger, SlideRightArranger, SlideTopArranger, ViewManager} from '@enact/ui/ViewManager';
+import {useCallback, useState} from 'react';
+
+import css from './ViewManager.module.less';
+
+const ViewManagerConfig = mergeComponentMetadata('ViewManager', ViewManager);
+
+ViewManagerConfig.defaultProps = {
+	duration: 300,
+	enteringDelay: 0
+};
+
+const prop = {
+	arranger: ['SlideBottomArranger', 'SlideLeftArranger', 'SlideRightArranger', 'SlideTopArranger', 'SlideArranger', 'CustomArranger (FadeAndSlideArranger)'],
+	direction: ['bottom', 'left', 'right', 'top']
+};
+
+const arrangers = {
+	'SlideBottomArranger': SlideBottomArranger,
+	'SlideLeftArranger': SlideLeftArranger,
+	'SlideRightArranger': SlideRightArranger,
+	'SlideTopArranger': SlideTopArranger
+};
+
+const views = new Array(10).fill().map((i, index) => {
+	return {
+		title: `Item ${index + 1}`,
+		content: `This is View ${index + 1}`
+	};
+});
+
+function ViewManagerLayout (props) {
```

Comment:
It's okay but let's make this an arrow function.

Reply:
I've changed this to an arrow function.